tests/integration/bigquery.spec.ts

```diff
@@ -360,6 +360,29 @@ suite("@dataform/integration/bigquery", ({ before, after }) => {
       expect(increment[increment.length - 2].statement).to.equal(table.postOps[0]);
       expect(increment[increment.length - 1].statement).to.equal(table.postOps[1]);
     });
+
+    test("contextual pre and post ops", async () => {
```

Comment:
i'm not sure this is the right place for these tests; seems like they should be going in `api.spec.ts`